mocha/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "mocha",
+  "version": "1.0.0",
+  "description": "Mocha API integration tests",
+  "scripts": {
+    "test": "mocha './mocha/test/**/*.js' --recursive --timeout 60000 --exit"
+  },
+  "private": true,
+  "devDependencies": {
+    "mocha": "^8.1.0"
+  }
```

Comment:
Missing dependencies, need to run npm install --save-dev should supertest chai http-status

Reply:
ok. I'll add. dependencies